SourceX/miniwin/misc_dx.cpp

```diff
@@ -20,8 +21,13 @@ WINBOOL SetCursorPos(int X, int Y)
 		X *= scaleX;
 		Y *= scaleX;
 	}
+#endif
 
+#ifdef USE_SDL1
+	SDL_WarpMouse(X, Y);
```

Comment:
Should be easy to wrap in `SDL_WarpMouseInWindow`?

Reply:
Done